app/components/UserValidator/index.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import * as React from 'react';
+import _ from 'lodash';
+import cx from 'classnames';
+import SearchPage from 'app/components/Search/SearchPage';
+import type { SearchResult } from 'app/reducers/search';
+import styles from './Validator.css';
+
+// $FlowFixMe
+import goodSound from '../../assets/good-sound.mp3';
+type State = {
+  showCompleted: boolean
+};
+type Props = {
```

Comment:
`location, onQueryChanged, results and searching` are not used in the `<Validator />` component. (i might be wrong here)
If this is the case, why isn't ESLint catching this?

Reply:
This and the `cx` are here because I'm lazy and just moved code from Abacard out to its own component without checking too much. I don't know why the linter don't catch these.

Reply:
Yehh i saw you made a `quick ctrl-c-v` on that cx(), but it was ugly and should be removed :)

Reply:
These props are given to the `SearchPage` component with `...this.props`.

Reply:
Yes the pris are passed on but I don't think the types are. So they should be typed in the next component?